js/common/components/images-to-gif.js

```diff
@@ -77,7 +77,7 @@ ImagesToGif.prototype.processImage = function (data, i, resolve, reject) {
     this.options.progressIndicator.setProgress(progress);
 
     var loadedImage = loadedImages[i];
-    if (i === loadedImages.length) {
+    if (i === loadedImages.length - 1) { // dunno why it needs since otherwise got empty frame?
```

Comment:
It's because the first frame is parsed out of this cycle.
Please remove that comment and just leave "-1"